lib/chef/user_v1.rb

```diff
@@ -93,11 +93,6 @@ def email(arg=nil)
                     arg, :kind_of => String)
     end
 
-    def password(arg=nil)
```

Comment:
I think I'm being dense.  Can someone point me at the password method that this is duplicating?